Vagrantfile

```diff
@@ -178,6 +178,19 @@ Vagrant.configure(2) do |config|
     config.vbguest.auto_update = true
   end
 
+  # confugre vagrant-proxyconf plugin
+  if Vagrant.has_plugin?('vagrant-proxyconf')
+    config.proxy.http     = ENV.fetch('http_proxy', '')
+    config.proxy.https    = ENV.fetch('https_proxy', '')
```

Comment:
Hmm I dont know if it will, Ill change it so that it only sets these variables if there are present in env. proxyconf probably only trys to set them if they are in config.proxy